testgrid/tgrun/pkg/scheduler/scheduler.go

```diff
@@ -264,12 +284,30 @@ func stringInSlice(s string, slice []string) bool {
 	return false
 }
 
-func bundleFromURL(url string) (string, error) {
+func getInstallerURL(url, kurlVersion string) (string, error) {
 	matches := urlRegexp.FindStringSubmatch(url)
 
 	if len(matches) != 3 {
 		return "", fmt.Errorf("unable to parse url %q", url)
 	}
 
-	return fmt.Sprintf("%s/bundle/%s.tar.gz", matches[1], matches[2]), nil
+	if kurlVersion == "" {
+		return url, nil
+	}
+
+	return fmt.Sprintf("%s/version/%s/%s", matches[1], kurlVersion, matches[2]), nil
```

Comment:
won't this result in including `version//` in URLs that didn't supply a kurlVersion?

Is that what we want to do?

Reply:
No cause line 294